### PR TITLE
fix: recoverdata and log print

### PR DESCRIPTION
**Pull request by vagetablechicken**
image

### CODE REVIEW

File: tools/openmldb_ops.py [hunk 1]
```diff
@@ -118,8 +118,8 @@ def RecoverPartition(executor, db, partitions, endpoint_status):
             db=db, table_name=table_name, pid=pid, leader_endpoint=leader_endpoint))
         status = executor.LoadTable(leader_endpoint, table_name, tid, pid)
         if not status.OK():
-            log.error("load table failed. db {db} name {table_name} tid {tid} pid {pid} endpoint {leader_endpoint} msg {status.GetMsg()}".format(
-                db=db, table_name=table_name, tid=tid, pid=pid, leader_endpoint=leader_endpoint, status=status))
+            log.error("load table failed. db {db} name {table_name} tid {tid} pid {pid} endpoint {leader_endpoint} msg {status}".format(
+                db=db, table_name=table_name, tid=tid, pid=pid, leader_endpoint=leader_endpoint, status=status.GetMsg()))
```
Comment (vagetablechicken) [inline tools/openmldb_ops.py:122]: LoadTable msg is not helpful, let me know why loadtable cmd RunWithRetuncode failed, `execute load table failed` is useless.

**Comment by vagetablechicken** [comment]
https://github.com/4paradigm/OpenMLDB/blob/a013ba33e4ce131353edc71e27053b1801ffb8f7/src/client/tablet_client.cc#L346

tablet client LoadTable should show error msg, next pr

**Comment by vagetablechicken** [comment]
recover may block by failed table, should try recover all, instead of return? done
 https://github.com/4paradigm/OpenMLDB/blob/0462f8a9682f8d232e8d44df7513cff66870d686/tools/openmldb_ops.py#L208

**Comment by vagetablechicken** [comment]
https://github.com/4paradigm/OpenMLDB/blob/9cc009114661d3cc80ff26a4c4705515870c17f9/src/tablet/tablet_impl.cc#L3006 nameserver say a replica should be on tablet, but tablet doesn't have the tid-pid dir on db_path, it's not a big problem, we just save metadata from loadtable request, but the code block and return. Just create one?
Next pr with table client error

File: tools/openmldb_ops.py [hunk 2]
```diff
@@ -205,7 +205,7 @@ def RecoverData(executor):
             return
         for name in tables:
             if not RecoverTable(executor, db, name).OK():
-                return
+                log.error(f"recover table failed. db {db} name {name}, check log for detail")
```
Comment (dl239) [inline tools/openmldb_ops.py]: f-string format is not supported in python2

Comment (vagetablechicken) [reply tools/openmldb_ops.py]: cvt to str format

File: tools/tool.py [hunk 3]
```diff
@@ -289,12 +291,12 @@ def LoadTable(self, endpoint, name, tid, pid, sync = True):
                     if table_stat == "kTableNormal":
                         return Status()
                     elif table_stat == "kTableLoading" or table_stat == "kTableUndefined":
-                        log.info("table is loading... tid {tid} pid {pid}".format(tid, pid))
+                        log.info("table is loading... tid {tid} pid {pid}".format(tid = tid, pid = pid))
                     else:
-                        return Status(-1, "table stat is {table_stat}".format(table_stat))
+                        return Status(-1, f"table stat is {table_stat}")
```
Comment (dl239) [inline tools/tool.py]: f-string format

**Review by dl239 — APPROVED** [review]

**Review by lqy222 — APPROVED** [review]

**Pull request merged**